src/containers/Transactions/simpleTab.scss

```diff
@@ -22,6 +22,8 @@ $subdued-color: $gray-400;
 
     .row {
       .value {
+        text-align: right;
```

Comment:
Why was this added?

Reply:
the text was being populated on the left side so I added this to format it correctly. Is there a better way to do this

Reply:
I will have to take a look